charts/opentelemetry-collector/Chart.yaml

```diff
@@ -12,3 +12,4 @@ maintainers:
   - name: TylerHelmuth
 icon: https://opentelemetry.io/img/logos/opentelemetry-logo-nav.png
 appVersion: 0.75.0
+kubeVersion: ">=1.24.0"
```

Comment:
We officially support only the minimum version the k8s team supports, which is currently 1.24.  It would be best not breaking users who want to use it on older versions of k8s, I just don't want to test on older versions, fix issues for older versions, or hold back features due to older versions.

Reply:
Understood - thanks for the discussion learned a few things. Will close the draft @TylerHelmuth @povilasv